src/metabase/search/config.clj

```diff
@@ -77,7 +77,7 @@
 (def models-search-order
   "The order of this list influences the order of the results: items earlier in the
   list will be ranked higher."
-  ["dashboard" "metric" "segment" "indexed-entity" "card" "dataset" "collection" "table" "action" "database"])
```

Comment:
This is also going to affect legacy search, but I think it's fine - quite a minor difference in the score anway.